modules/apps/headless/headless-builder/headless-builder-test/src/testIntegration/java/com/liferay/headless/builder/model/listener/test/APIEndpointRelevantObjectEntryModelListenerTest.java

```diff
@@ -141,6 +142,26 @@ public void test() throws Exception {
 			"Path can have a maximum of 255 alphanumeric characters.",
 			jsonObject.get("title"));
 
+		jsonObject = HTTPTestUtil.invokeToJSONObject(
+			JSONUtil.put(
+				"httpMethod", "get"
+			).put(
+				"name", RandomTestUtil.randomString()
+			).put(
+				"path",
+				StringBundler.concat(
+					RandomTestUtil.randomString(), StringPool.FORWARD_SLASH,
+					StringPool.COMMA)
+			).put(
+				"scope", "company"
+			).toString(),
+			"headless-builder/endpoints", Http.Method.POST);
+
+		Assert.assertEquals("BAD_REQUEST", jsonObject.get("status"));
```

Comment:
I think this assertion could be done easily with JSONAssert. Please have a look at other examples in this module, so I guess you can even include the assertion and the invocation inline

Reply:
Something like:

```
JSONAssert.assertEquals(
			JSONUtil.put(
				"status", BAD_REQUEST"
			).put(
				"title", ......
			[...]
				JSONUtil.put(
					JSONUtil.put(
						"relatedTextProperty1", JSONUtil.put("value2")
					).put(
						"relatedTextProperty2", JSONUtil.put("value3")
					).put(
						"textProperty", "value1"
					))
			).toString(),
			HTTPTestUtil.invokeToJSONObject(
				..........................
			).toString(),
			JSONCompareMode.STRICT);
```